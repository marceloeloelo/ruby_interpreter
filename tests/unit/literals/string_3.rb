puts('foo#{x}bar')
puts("hola")
#puts("foo#{}bar") no se soporta...
x = "aux"
puts("foo #{ x } bar")
puts("foo #{x} bar")
puts("foo #{x } bar")
puts("foo #{x y} bar")
