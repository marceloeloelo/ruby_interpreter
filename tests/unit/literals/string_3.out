foo#{x}bar
hola
foo aux bar
foo aux bar
foo aux bar
InterpolarionError: more than one variable is not supported
